### PR TITLE
Split syntax convetion doc into multiple files for better display

### PR DESCRIPTION
**Pull request by lancelly**

### CODE REVIEW

File: docs/zh/UserGuide/Reference/Keywords.md [hunk 1]
```diff
@@ -19,9 +19,15 @@
 
 -->
 
-# 关键字和保留字
+# 关键字
 
-以下为 IoTDB 的关键字，其中保留字用 **(R)** 标志。
+以下为 IoTDB 的保留字：
```
Comment (qiaojialin) [inline docs/zh/UserGuide/Reference/Keywords.md:24]: 保留字

File: docs/zh/UserGuide/Reference/Keywords.md [hunk 2]
```diff
@@ -19,9 +19,15 @@
...
+
+- ROOT
+- TIME
+- TIMESTAMP
+
+以下为 IoTDB 的关键字：
 
 - 一般关键字
```
Comment (qiaojialin) [inline docs/zh/UserGuide/Reference/Keywords.md:32]:
```suggestion
一般关键字
```

File: docs/UserGuide/Reference/Keywords.md [hunk 3]
```diff
@@ -21,7 +21,13 @@
 
 # Keywords and Reserved Words
 
-The following list shows the keywords and reserved words in IoTDB. Reserved keywords are marked with **(R)**.
+The following list shows the reserved words in IoTDB:
```
Comment (qiaojialin) [inline docs/UserGuide/Reference/Keywords.md:24]:
```suggestion
Reserved words
```

File: docs/UserGuide/Reference/Keywords.md [hunk 4]
```diff
@@ -21,7 +21,13 @@
...
+
+- ROOT
+- TIME
+- TIMESTAMP
+
+The following list shows the keywords in IoTDB:
 
 - Common Keywords
```
Comment (qiaojialin) [inline docs/UserGuide/Reference/Keywords.md:32]:
```suggestion
Common Keywords

```

File: docs/zh/UserGuide/Reference/Keywords.md (hunk 1 above)
Comment (lancelly) [reply docs/zh/UserGuide/Reference/Keywords.md]: Done.

File: docs/zh/UserGuide/Reference/Keywords.md (hunk 2 above)
Comment (lancelly) [reply docs/zh/UserGuide/Reference/Keywords.md]: Done.

File: docs/UserGuide/Reference/Keywords.md (hunk 3 above)
Comment (lancelly) [reply docs/UserGuide/Reference/Keywords.md]: Done.

File: docs/UserGuide/Reference/Keywords.md (hunk 4 above)
Comment (lancelly) [reply docs/UserGuide/Reference/Keywords.md]: Done.

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**